packages/components/src/components/assay/AssayResultDeleteModal.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import {AssayResultDeleteModal} from "./AssayResultDeleteModal";
+import {SchemaQuery, SCHEMAS} from "../..";
+
+describe('<AssayResultDeleteModal/>', () => {
+    test('Show progress', () => {
+
+        const schemaQuery = SchemaQuery.create(SCHEMAS.EXP_TABLES.ASSAY_RUNS.getSchema(), SCHEMAS.EXP_TABLES.ASSAY_RUNS.getQuery());
+
+        const component = (
+            <AssayResultDeleteModal
+                onCancel={jest.fn()}
+                afterDelete={jest.fn()}
+                afterDeleteFailure={jest.fn()}
+                schemaQuery={schemaQuery}
```

Comment:
You can just set `schemaQuery={SCHEMAS.EXP_TABLES.ASSAY_RUNS}` as it is already a `SchemaQuery`.

Reply:
Good call.  Applied from your patch